packages/odyssey-storybook/src/utilities/Utilities.stories.mdx

```diff
@@ -0,0 +1,202 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Utilities/Utilities" />
+
+# Utilities
+
+## Box VS Text
+
+In production docs, these sections will live with their appropriate utility. I've co-located them here, so that we can align internally.
+
+### Box
+
+`<Box />` acts as the root for most components. Prefer spreading omitted rest props and forwarding a ref to `<Box />` as well.
+
+This component uses an `as="abbr"` polymorphic API to keep the DOM a11y compliant as well as clean and comprehensible by developers and their automated tests.
+
+`<Box />` contains a minimal set of styles that are provided through inheritance to children (`font-family`) and provides an API for utility tailwind-esque styling (`padding`, `margin`).
+
+These styles that are owned by `<Box />` are also a centralized place to apply theme-able odyssey design tokens for override (theming `font-family` "globally").
+
+Going further, `<Box />` accepts a `className` prop for applying component-specific styling. This prevents the need for `<Box />` to define every style for all use cases. Additionally, parent component classes can contain their own theme-able tokens to keep things customizable.
```

Comment:
I would omit some of these implementation details (`className` bit etc) until our implementation has matured - see more in my comments below. Lets keep the writeup here high level for now

Reply:
Def agreed here as well. Wanted to make sure we're internally aligned, but none of these workings should be surfaced until the components are solid.